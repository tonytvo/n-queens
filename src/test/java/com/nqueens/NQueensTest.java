package com.nqueens;

import com.google.common.collect.ImmutableSet;
import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

import java.util.Set;

public class NQueensTest {

    @Test
    public void foo() {
        NQueens nQueens = new NQueens();

        Set<QueenPosition> queenPositions = nQueens.findQueenPositions(4);

        Approvals.verify(queenPositions);
    }
}
