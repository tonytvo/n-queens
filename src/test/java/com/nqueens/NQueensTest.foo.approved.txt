[{[{a, 128}, {b, 126}, {c, 129}, {d, 127}]}, {[{a, 127}, {b, 129}, {c, 126}, {d, 128}]}]